tests/Zend/Session/SessionTest.php

```diff
@@ -1133,7 +1133,7 @@ public function testInvalidPreexistingSessionIdDoesNotPreventRegenerationOfSid()
         session_start();
 
         // Set session ID to invalid value
-        session_id('xxx');
+        @session_id('xxx');
```

Comment:
What was the warning here?

Reply:
I found it, in the build logs:

> session_id(): Cannot change session id when session is active

I'd argue that this test isn't actually testing what it is intending, since it's verifying that the session isn't "xxx", but it wouldn't possibly be since this session_id call failed. It looks like maybe we should move the session_id call before session_start, from http://php.net/session_id:

> If id is specified, it will replace the current session id. session_id() needs to be called before session_start() for that purpose. Depending on the session handler, not all characters are allowed within the session id. For example, the file session handler only allows characters in the range a-z A-Z 0-9 , (comma) and - (minus)!

Reply:
I tried that, throws a different warning on `session_start`:

>session_start(): open(/path/to/library/zf1-session/tests/Zend/Session/_files/ZF-3378/x/sess_xxx, O_RDWR) failed: No such file or directory (2)

Reply:
Earlier in the test, it's explicitly creating directories in the `ZF-3378` directory, but only `[0-9a-v]`. Which is why I think `xxx` was chosen as the sid, since it would be invalid (no `x` directory for it to write to).

I can mess around with it some more.